gem/templates/profiles/viewprofile.html

```diff
@@ -4,7 +4,7 @@
 {% block content %}
     <div class="profile-container">
         <div class="title profile">
-            <h3>{% trans "Hello " %}{{ user }}</h3>
+            <h3>{% trans "Hello " %} {{ user }}</h3>
```

Comment:
yeah, lets remove the space from the translation. It should just be `{% trans "Hello" %}`